core/ts/utils/utils.ts

```diff
@@ -100,6 +177,13 @@ const unpackTallyVotesSmallVals = (packedVals: bigint) => {
   return { numSignUps, batchStartIndex };
 };
 
+/**
+ * This function packs it's parameters into a single bigint.
+ * @param row - The row.
+ * @param col - The column.
+ * @param numSignUps - The number of sign ups.
+ * @returns Returns a single bigint that contains the packed values.
+ */
 const packSubsidySmallVals = (row: number, col: number, numSignUps: number) => {
```

Comment:
do we have an unpack function for subsidy values?

Reply:
there's no need to have it